src/utils/validators.js

```diff
@@ -166,17 +166,17 @@ const INFOCOLS_INFOBOX_DESCRIPTION_MAX_LENGTH = 120
 // ===============
 // Contacts
 const CONTACT_TITLE_MIN_LENGTH = 1
-const CONTACT_TITLE_MAX_LENGTH = 30
+const CONTACT_TITLE_MAX_LENGTH = 50
 const CONTACT_DESCRIPTION_MAX_LENGTH = 400
 
 // Locations
 const LOCATION_TITLE_MIN_LENGTH = 1
-const LOCATION_TITLE_MAX_LENGTH = 30
-const LOCATION_ADDRESS_MAX_LENGTH = 30
+const LOCATION_TITLE_MAX_LENGTH = 50
+const LOCATION_ADDRESS_MAX_LENGTH = 50
 const LOCATION_OPERATING_DAYS_MIN_LENGTH = 2
-const LOCATION_OPERATING_DAYS_MAX_LENGTH = 30
+const LOCATION_OPERATING_DAYS_MAX_LENGTH = 50
 const LOCATION_OPERATING_HOURS_MIN_LENGTH = 2
-const LOCATION_OPERATING_HOURS_MAX_LENGTH = 30
+const LOCATION_OPERATING_HOURS_MAX_LENGTH = 50
```

Comment:
ok on the frontend - but are these also coded into our backend?

Reply:
No changes required for backend! Backend checks for type of input, but not the length - I think we're okay with this because it's more of a restriction to help users not misuse the fields - but Shaz was showing use cases where the current limit is insufficient